index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<!--This means that the browser will (probably) render the width of the page at the width of its own screen-->
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Portafolio Page</title>
+	<!-- Call File CSS-->
+	<link rel="stylesheet" href="css/style.css">
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+</head>
+<body>
+	<div class="container">
+	<!-- Header that contains the Logo and Main Title -->
+		<header class="header">
+			<div class="Logo-Responsive">
+				<img class="img-responsive" src="Images/LogoUdacity.png" alt="Logo">
+			</div>
+			<div class="header-text">
+				<h1 class="header-title">Marcos Rodriguez</h1>
+				<h4 class="header-subtitle">Frontend Ninja</h4>
+			</div>			
+		</header>
+		<!-- Figure that contains the dividing line in the webpage -->
+		<figure class="figure-line">
+			<hr>
+		</figure>
+		<!-- Figure that contains the center image-->
+        <figure>
```

Comment:
#### Nitpick :warning: 
You don’t have to forget about the use of indentations because it makes your code more understandable.